examples/pom.xml

```diff
@@ -166,6 +156,11 @@
               <artifactId>java-driver-mapper-processor</artifactId>
               <version>${project.version}</version>
             </path>
+            <path>
+              <groupId>com.scylladb</groupId>
+              <artifactId>java-driver-mapper-runtime</artifactId>
+              <version>${project.version}</version>
+            </path>
           </annotationProcessorPaths>
```

Comment:
Why is this needed? This is compiler's configuration section for annotation processors. Is the runtime needed here?

Reply:
it is required, otherwise it will fail to compile it.